bench/config/lets_encrypt.py

```diff
@@ -105,20 +105,10 @@ def create_dir_if_missing(path):
 		os.makedirs(os.path.dirname(path))
 
 
-def get_certbot():
-	from urllib.request import urlretrieve
-
-	certbot_path = get_certbot_path()
-	create_dir_if_missing(certbot_path)
-
-	if not os.path.isfile(certbot_path):
-		urlretrieve("https://dl.eff.org/certbot-auto", certbot_path)
-		os.chmod(certbot_path, 0o744)
-
-
```

Comment:
Check if certbot is installed and prompt the user to install it following some ABCs. Perhaps a link to a guide on Frappe Docs